packages/ipfs-core/package.json

```diff
@@ -122,6 +122,7 @@
     "uint8arrays": "^2.1.3"
   },
   "devDependencies": {
+    "libp2p-interfaces": "^0.8.4",
```

Comment:
Otherwise

```
src/ipns/routing/config.js:26:36 - error TS2345: Argument of type 'import("/Users/gozala/Projects/js-ipfs-2/node_modules/libp2p/node_modules/libp2p-interfaces/dist/src/pubsub/index")' is not assignable to parameter of type 'import("/Users/gozala/Projects/js-ipfs-2/node_modules/libp2p-interfaces/dist/src/pubsub/index")'.
  Types of property 'peers' are incompatible.
    Type 'Map<string, import("/Users/gozala/Projects/js-ipfs-2/node_modules/libp2p/node_modules/libp2p-interfaces/dist/src/pubsub/peer-streams")>' is not assignable to type 'Map<string, import("/Users/gozala/Projects/js-ipfs-2/node_modules/libp2p-interfaces/dist/src/pubsub/peer-streams")>'.
      Type 'import("/Users/gozala/Projects/js-ipfs-2/node_modules/libp2p/node_modules/libp2p-interfaces/dist/src/pubsub/peer-streams")' is not assignable to type 'import("/Users/gozala/Projects/js-ipfs-2/node_modules/libp2p-interfaces/dist/src/pubsub/peer-streams")'.
        Types have separate declarations of a private property '_rawOutboundStream'.
26     pubsubDs = new PubsubDatastore(pubsub, localDatastore, peerId)
```